docs/changelog.rst

```diff
@@ -31,6 +40,16 @@ Bugs fixed
    input, rather than silenltly producing nonsense.
  * :func:`Layout.parse_multivector` supports basis vector names which do not
    start with e.
+ * :func:`clifford.tools.g3c.val_midpoint_between_lines` now handles the case that
+   the two lines are touching.
+ * :func:`clifford.tools.g3c.val_fit_circle` now correctly selects the first and
+   second eigenvalue regardless of order.
+ * :func:`clifford.tools.g3c.sphere_beyond_plane` now tested and correct.
+ * :func:`clifford.tools.g3c.sphere_behind_plane` now tested and correct.
+ * :func:`clifford.tools.g3c.val_unsign_sphere` is now jitted, as it should have
+   been from the start.
+ * :func:`clifford.tools.g3c.get_nearest_plane_point` correctly returns the conformal
+   point rather than the 3D point.
```

Comment:
```suggestion
 * :mod:`clifford.tools.g3c`:
   * :func:`~clifford.tools.g3c.val_midpoint_between_lines` now handles the case that
     the two lines are touching.
   * :func:`~clifford.tools.g3c.val_fit_circle` now correctly selects the first and
     second eigenvalue regardless of order.
   * :func:`~clifford.tools.g3c.sphere_beyond_plane` now tested and correct.
   * :func:`~clifford.tools.g3c.sphere_behind_plane` now tested and correct.
   * :func:`~clifford.tools.g3c.val_unsign_sphere` is now jitted, as it should have
     been from the start.
   * :func:`~clifford.tools.g3c.get_nearest_plane_point` correctly returns the conformal
     point rather than the 3D point.
```